scout/load/panel.py

```diff
@@ -151,6 +174,48 @@ def load_panelapp_panel(adapter, panel_id=None, institute="cust000", confidence=
             raise err
 
 
+def load_panelapp_green_panel(adapter, institute, force):
+    """Load/Update the panel containing all Panelapp Green genes
+
+    Args:
+        adapter(scout.adapter.MongoAdapter)
+        institute(str): _id of an institute
+        force(bool): force update panel even if it has fewer genes than previous version
+    """
+    LOG.info("Fetching all panel app panels")
+    panel_ids = _panelapp_panel_ids()
+
+    # check and set panel version
+    old_panel = adapter.gene_panel(panel_id=PANEL_NAME)
+    green_panel = {
+        "panel_name": PANEL_NAME,
+        "display_name": "PanelApp Green Genes",
+        "institute": institute,
+        "version": old_panel["version"] + 0.1 if old_panel else 1,
```

Comment:
This increment of version by 0.1 is something that doesn't really work as one would expect. After the first couple of updates you get this:

<img width="953" alt="image" src="https://user-images.githubusercontent.com/28093618/217455662-d81abc35-a49c-4f42-b0fe-c8c8da1c6874.png">

Which is due to this:

https://stackoverflow.com/questions/11640902/incremented-floats-do-not-equal-each-other

I'll fix

Reply:
Nice, I didn't think about that! We should have a structured type there really - I mean, it isn't strictly speaking a float, it is more like a tuple of ints.

Reply:
I'm copying the code we use for the OMIM panel: https://github.com/Clinical-Genomics/scout/blob/a998f0941583cd64b9e495319e22f1532d62c850/scout/adapter/mongo/panel.py#L67

Reply:
Also nice in its own right that they increment in the same way.

Reply:
> Also nice in its own right that they increment in the same way.

yes!